apdu-core/src/command.rs

```diff
@@ -93,7 +101,15 @@ impl From<Command> for Vec<u8> {
         }
 
         if let Some(l) = le {
-            buffer.push(l);
+            if l > u8::MAX.into() {
```

Comment:
As the final step, could you please add `cfg!(feature = "longer_payloads")` into the condition to avoid an unexpected behaviour changes as same as L92?